frontend/src/services/ErrorHandler.test.ts

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+import * as React from 'react'
+
+jest.mock('./ErrorHandler')
+
+describe('errorHandler', () => {
+    it ('renders toast if error occurs', () => {
```

Comment:
Considering code you have, it will be expected to see next tests:
 - Call errorHandler with correct error (that fulfills IError interface)
 - Case when `destructuredMessage` does not contain any message
 - Case when error aren't fulfills IError interface

Reply:
no understand how to make tests for this